src/bundle.rs

```diff
@@ -66,37 +70,65 @@ pub struct Flags {
     /// guaranteed to be dummy notes. If `true`, the created notes may be either real or
     /// dummy notes.
     outputs_enabled: bool,
+    /// Flag denoting whether ZSA transaction is enabled.
+    ///
+    /// If `false`,  all notes within [`Action`]s in the transaction's [`Bundle`] are
+    /// guaranteed to be notes with native asset.
```

Comment:
_From str4d ([r2585536670](https://github.com/zcash/orchard/pull/471#discussion_r2585536670))_

```suggestion
    /// Flag denoting whether ZSA functionality is enabled in the transaction.
    ///
    /// If `false`, all notes within [`Action`]s in the transaction's [`Bundle`] are
    /// guaranteed to be notes with native asset. If `true`, `Action`s may use any asset.
```

Reply:
FIXED